node/src/tests/test-Producer.ts

```diff
@@ -235,6 +235,69 @@ test('transport2.produce() succeeds', async () =>
 			});
 }, 2000);
 
+test.only('transport1.produce() without header extensions and rtcp succeeds', async () =>
```

Comment:
No similar test for Rust?

Reply:
No need because in Rust all those three types are not optional but mandatory and all have default values already.